master/internal/api_workspace.go

```diff
@@ -830,11 +845,10 @@ func (a *apiServer) setWorkspaceNamespaceBindings(ctx context.Context,
 ) (resp *apiv1.SetWorkspaceNamespaceBindingsResponse, err error) {
 	defer func() {
 		if v := recover(); v != nil {
-			err = status.Error(codes.InvalidArgument, "Auto-creating namespaces is "+
+			err = status.Errorf(codes.InvalidArgument, "Auto-creating namespaces is "+
```

Comment:
does ```Error``` vs ```Errorf``` here matter since we aren't specifying formatting args?

Reply:
Good catch, changed!